user-console/src/main/java/org/pentaho/mantle/client/MantleUtils.java

```diff
@@ -94,6 +94,10 @@ public boolean checkSelectedPerspective() {
   public static native String getSchedulerPluginContextURL()/*-{
     if (typeof $wnd.pho.getSchedulerPluginContextURL !== "undefined" ) {
       return $wnd.pho.getSchedulerPluginContextURL();
+    } else {
+      // fallback to the default location when the getter is unavailable
+      // e.g. race condition loading GWT JavaScript code, PUC JS executes before the Scheduler Plugin JS is loaded
+      return $wnd.location.protocol + "//" + $wnd.location.host + $wnd.CONTEXT_PATH + "plugin/scheduler-plugin/";
```

Comment:
`FULL_QUALIFIED_URL` should also be available to use here. please use it instead, as it is equivalent to `$wnd.location.protocol + "//" + $wnd.location.host + $wnd.CONTEXT_PATH`